src/components/App.js

```diff
@@ -8,20 +8,19 @@ class App extends React.Component {
   this.createNewEvent = this.createNewEvent.bind(this)
 
   this.state = {
-    urlToShare: ""
+    urlToShare: "", //populated by createNewEvent when form is submitted
+    formData: {},   //populated by form submit
```

Comment:
I've called this eventData... and it's passed down to the CreateEvent component, so can we use eventData please to save a load of refactoring